modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/Oauth2PersistenceProcessorTestCase.java

```diff
@@ -48,21 +53,24 @@ public void testInit() throws Exception {
     public void atEnd() throws Exception {
 
         resetISConfiguration();
+        restClient.closeHttpClient();
     }
 
     @Test(groups = "wso2.is", description = "Test PlainTextPersistenceProcessor")
     public void testPlainTextPersistenceProcessor() throws Exception {
 
-        OAuthConsumerAppDTO oAuthConsumerAppDTO = createApplication("app1");
-        Assert.assertNotNull(oAuthConsumerAppDTO, "Application creation failed.");
+        ApplicationResponseModel application = createApplication("app1");
+        Assert.assertNotNull(application, "Application creation failed.");
+        String applicationId1 = application.getId();
 
-        consumerKey1 = oAuthConsumerAppDTO.getOauthConsumerKey();
+        OpenIDConnectConfiguration oidcConfig = getOIDCInboundDetailsOfApplication(applicationId1);
+        String consumerKey1 = oidcConfig.getClientId();
         Assert.assertNotNull(consumerKey1, "Application creation failed.");
 
-        String consumerSecret1 = oAuthConsumerAppDTO.getOauthConsumerSecret();
+        String consumerSecret1  = oidcConfig.getClientSecret();
```

Comment:
```suggestion
        String consumerSecret1 = oidcConfig.getClientSecret();
```

Reply:
Fixed with https://github.com/wso2/product-is/pull/16271/commits/e9343598362e5021e1f6832b61ef1997495afeda